### PR TITLE
Add documentation on changing address spaces

### PR DESCRIPTION
**Pull request by lulf**

### CODE REVIEW

File: documentation/assemblies/assembly-managing-address-spaces.adoc [hunk 1]
```diff
@@ -29,9 +35,9 @@ include::../common/address-space-example2.yaml[]
...
 <3> The authentication service type can be `none`, `standard` or `external`. External requires an additional field, `details`, including `host` and `port` information.
-<4> The endpoints that must be configured for the `AddressSpace`.  Endpoints can specity either `messaging`, `console` or `mqtt` services. However, the `mqtt` service is supported for the `standard` address space type only.
-<5> Expose the configuration of an endpoint. Endpoints can be kept internal to the cluster, or exposed using an OpenShift route or a load-balancer service.
-<6> The certificate configuration can be used to specify how the endpoint certificate is provided.  The provider can be `selfsigned` (default), `wildcard` (requires it to be enabled by the {ProductName} admin), `openshift` (signed by the OpenShift cluster CA), or `certBundle` (base64-encoded PEM key and certificate).
+<4> The endpoints that must be configured for the `AddressSpace`.  Endpoints can specity either `messaging`, `console` or `mqtt` services. However, the `mqtt` service is supported for the `standard` address space type only. Endpoints may be changed by replacing the address space.
```
Comment (jenmalloy) [inline documentation/assemblies/assembly-managing-address-spaces.adoc]: Change "may" to "can"

File: documentation/assemblies/assembly-managing-address-spaces.adoc [hunk 2]
```diff
@@ -29,9 +35,9 @@ include::../common/address-space-example2.yaml[]
...
+<5> Expose the configuration of an endpoint. Endpoints can be kept internal to the cluster, or exposed using an OpenShift route or a load-balancer service. Expose configuration may be changed by replacing the address space.
```
Comment (jenmalloy) [inline documentation/assemblies/assembly-managing-address-spaces.adoc]: Change "may" to "can"

File: documentation/assemblies/assembly-managing-address-spaces.adoc [hunk 3]
```diff
@@ -5,7 +5,13 @@
 [id='managing-address-spaces-{context}']
 = Managing address spaces
 
-{ProductName} is configured to support managing address spaces using the {KubePlatform} command-line tools.
+{ProductName} is configured to support managing address spaces using the {KubePlatform} command-line
+tools. Address spaces are managed like any other {KubePlatform} resource using `{cmdcli}`.
+
+Address spaces may also be replaced in order to change the plan, endpoints, network policies or
```
[the remark this answers is not in the document]
Comment (lulf) [reply documentation/assemblies/assembly-managing-address-spaces.adoc]: Fixed

File: documentation/assemblies/assembly-managing-address-spaces.adoc (hunk 2 above)
Comment (lulf) [reply documentation/assemblies/assembly-managing-address-spaces.adoc]: Fixed

**Comment by lulf** [comment]
@jenmalloy Sorry I forgot to add one file, added now.

**Pull request merged**